discovery/syncer.go

```diff
@@ -450,6 +450,8 @@ func (g *GossipSyncer) Start() {
 // exited.
 func (g *GossipSyncer) Stop() {
 	g.stopped.Do(func() {
+		log.Debugf("Stopping GossipSyncer(%x)", g.cfg.peerPub[:])
```

Comment:
nit: Add a defer with "stopped" message here too just for completeness?

Reply:
Updated!